website/views/options/social/boss.jade

```diff
@@ -5,23 +5,50 @@ mixin boss(tavern, mobile)
       h3.panel-title(ng-if='group.quest.active==false')=env.t('questInvitation') + ' {{::Content.quests[group.quest.key].text()}}'
       h3.panel-title(ng-if='group.quest.active==true') {{::Content.quests[group.quest.key].text()}}
     .panel-body.modal-fixed-height
-      div(ng-if='group.quest.active==false')
-        table.table.table-striped
-          tr(ng-repeat='member in group.members')
-            td
-              span(ng-if=':: group.quest.leader && group.quest.leader==member._id && isMemberOfGroup(group.quest.leader,group) && isMemberOfPendingQuest(group.quest.leader,group)')
-                |*&nbsp;
-              |{{::member.profile.name}}
-            td {{group.quest.members[member._id] === true ? env.t('accepted') : group.quest.members[member._id] === false ? env.t('rejected') : env.t('pending')}}
+      div(ng-if='group.quest.active==false' ng-init="selectedTab = 1")
+        ul.nav.nav-pills
+          li(ng-class="{active:selectedTab === 1}")
+            a(ng-click="selectedTab = 1")
+                =env.t('invitations')
+          li(ng-class="{active:selectedTab === 2}")
+            a(ng-click="selectedTab = 2")
+              =env.t('questDetails')
+        div(ng-show="selectedTab === 1")
+          br
+          table.table.table-striped(bindonce='group')
+            tr(ng-repeat='member in group.members track by member._id')
+              td.media
+                span(ng-if=':: group.quest.leader && group.quest.leader==member._id && isMemberOfGroup(group.quest.leader,group) && isMemberOfPendingQuest(group.quest.leader,group)') 
+                  |*&nbsp;
+                a.media-body
```

Comment:
Nothing! I copied the code from [here](https://github.com/HabitRPG/habitrpg/blob/develop/website/views/options/social/group.jade#L80-87), and forgot to remove them (they're necessary in the original code, but not here). Good catch.